airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java

```diff
@@ -372,6 +378,14 @@ public static void main(final String[] args) throws IOException, InterruptedExce
         workspaceHelper,
         workerConfigs);
 
+    final JobNotifier jobNotifier = new JobNotifier(
+        configs.getWebappUrl(),
+        configRepository,
+        new WorkspaceHelper(configRepository, jobPersistence),
+        TrackingClientSingleton.get());
```

Comment:
I'm not seeing this be initialised? Am I missing something?

Reply:
It is initialized in the get method
```
public static TrackingClient get() {
    synchronized (lock) {
      if (trackingClient == null) {
        initialize();
      }
      return trackingClient;
    }
  }
```

Reply:
ah I was referring to properly initialising it with this call https://github.com/airbytehq/airbyte/blob/master/airbyte-analytics/src/main/java/io/airbyte/analytics/TrackingClientSingleton.java#L39. e.g. https://github.com/airbytehq/airbyte/blob/master/airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java#L259

I think this is needed otherwise we won't be sending events to Segment. If you look at the Scheduler code, the linked code is called before we call `get`. I'm assuming we want to do the same here.

Reply:
Thanks for catching it, Done